scripts/jenkins-yoctobuild-build.sh

```diff
@@ -82,6 +82,12 @@ EOF
 # Build Go repositories.
 export GOPATH="$WORKSPACE/go"
 for build in deployments deviceadm deviceauth inventory useradm; do (
+
+    # If we are testing a specific microservice, only build that one.
+    if [[ ! -z $MICROSERVICE_TO_TEST && $build != $MICROSERVICE_TO_TEST ]]; then
+        continue
+    fi
+
```

Comment:
Btw, I also noticed that GUI and API gateway are not covered here (see below). Intentional?

Probably it doesn't hurt to just build them every time, but maybe you don't have the Git repo cloned at this point.